core/src/main/resources/hudson/model/Api/index.jelly

```diff
@@ -98,58 +98,47 @@ THE SOFTWARE.
         -->
       </dl>
 
-      <p>
-        For more information about remote API in Jenkins, see
-        <a href="https://jenkins.io/redirect/remote-api">the documentation</a>.
-      </p>
+      <p>For more information about remote API in Jenkins, see <a href="https://jenkins.io/redirect/remote-api">the documentation</a>.</p>
```

Comment:
Just please avoid touching unrelated lines of code when filing a PR. The longer the diff, the harder it is to review, thus the less likely it is to be merged (and the more likely your change is to cause problems for merging other changes, cherry-picking fixes to LTS, `blame`ing, etc.).